src/main/services/__tests__/ServiceContainer.test.ts

```
@@ -53,52 +76,49 @@ describe('ServiceContainer', () => {
-    consoleLog = jest.spyOn(console, 'log').mockImplementation();
-    consoleError = jest.spyOn(console, 'error').mockImplementation();
-    consoleWarn = jest.spyOn(console, 'warn').mockImplementation();
+    consoleLog = spyOn(console, 'log').mockImplementation(() => {});
+    consoleError = spyOn(console, 'error').mockImplementation(() => {});
+    consoleWarn = spyOn(console, 'warn').mockImplementation(() => {});
 
     // Mock DatabaseManager
-    (DatabaseManager as jest.MockedClass<typeof DatabaseManager>).mockImplementation(() => ({
-      initialize: jest.fn(),
-    } as any));
+    (DatabaseManager as any) = mock(() => ({
```

Biome (2.1.2)

[error] The imported variable DatabaseManager is read-only

The variable is imported here

Use a local variable instead of reassigning an import.

(lint/suspicious/noImportAssign)